index.js

```diff
@@ -131,6 +150,13 @@ module.exports = function (session) {
    */
 
   PGStore.prototype.query = function (query, params, fn) {
+    if (this.pgPool !== undefined) {
+      this.pgPool.query(query, params, function (err, res) {
```

Comment:
There's a difference how the `params` argument is handled here: To match the prior behavior, do: `params || []`